lmdeploy/lite/quantization/awq.py

```diff
@@ -66,7 +66,8 @@ def get_weight_scale(weight, q_group_size=-1):
     org_shape = weight.shape
     if q_group_size > 0:
         weight = weight.view(-1, q_group_size)
-    scale = weight.abs() / weight.abs().amax(dim=1, keepdim=True)
+    scale = weight.abs() / weight.abs().amax(dim=1,
+                                             keepdim=True).clamp(min=1e-4)
```

Comment:
Do we need to enable the default `clamp`? Will it affect the accuracy? When users encounter NaN issues with the default `auto_awq`, would it be better to enable this method through parameters later on?

Reply:
These are also what I want to discuss here. The PR status is a draft now.